aat/src/aat/resources/features/F-099 SearchParty (GlobalSearch)/F-099.feature

```diff
@@ -0,0 +1,144 @@
+@F-099
+Feature: F-099: Search Party (Global Search)
+
+  Background:
+    Given an appropriate test context as detailed in the test data source
+
+  @S-099.1 @AC-1 @AC-2 @AC-3 @AC-4 @AC-5 @AC-6 @AC-7 @AC-8 @AC-9 @AC-10 @AC-11 @AC-12 @AC-13 @AC-25 @AC-26 @AC-27 @AC-28 @AC-29
+  Scenario: Import Definition file with correctly configured, single case field name in SearchPartyName column of SearchParty tab
+    Given a user with [an active profile in CCD]
+    When a request is prepared with appropriate values
+    And it is submitted to call the [Import definition file] operation of [CCD Definition Store]
+    Then a positive response is received
+    And the response has all other details as expected
+    And a call [fetch Complex Fields Ordering Schema to verify single case field values in columns SearchPartyName(AC1), SearchPartyEmailAddress(AC6), SearchPartyAddressLine1(AC8), SearchPartyPostCode(AC10) and SearchPartyDoB(AC12)] will get the expected response as in [S-099.1_Get_ComplexFieldsOrdering].
+    And a call [fetch Multiple Pages Case Type Schema to verify comma separated case field names in SearchPartyName column(AC5)] will get the expected response as in [S-099.1_Get_MultiplePagesCaseType].
+    And a call [fetch Conditionals Case Type Schema to verify complex field name in columns SearchPartyName(AC3), SearchPartyEmailAddress(AC7), SearchPartyAddressLine1(AC9), SearchPartyPostCode(AC11), SearchPartyDob(AC13)] will get the expected response as in [S-099.1_Get_ConditionalsCaseType]
+    And a call [fetch Retain Hidden Value Case Type Schema to verify comma separated complex field names in SearchPartyName column(AC4) and comma separated combination of top level field names and complex field names in SearchPartyName column(AC5)] will get the expected response as in [S-099.1_Get_RetainHiddenValueCaseType]
+    And a call [fetch Tabs Case Type Schema to verify blank value in the columns SearchPartyName(AC25), SearchPartyEmailAddress(AC26), SearchPartyAddressLine1(AC27), SearchPartyDoB(AC28) and SearchPartyPostCode(AC29)] will get the expected response as in [S-099.1_Get_TabsCaseType]
+
+  @S-099.2 @AC-14
+  Scenario: Import Definition file with in-correct value in "CaseTypeId" column of the SearchParty tab
+    Given a user with [an active profile in CCD]
+    When a request is prepared with appropriate values
+    And the request [contains an invalid value (not matching the CaseType tab) in the configured 'CaseTypeId' column of the SearchParty tab]
+    And it is submitted to call the [Import definition file] operation of [CCD Definition Store]
+    Then a negative response is received
+    And the response has all other details as expected
+    And a call [to fetch incorrect caseTypeId schema] will get the expected response as in [S-099.2_Get_InvalidCaseType].
+
+  @S-099.3 @AC-15
+  Scenario: Import Definition file with invalid complex case field name in the "SearchPartyDoB" column of the SearchCriteria tab
```

Comment:
Did you mean search criteria or search party tab?

Reply:
Fixed in https://github.com/hmcts/ccd-definition-store-api/pull/1059/commits/d41404b1f9cf0243dc133d9bf075292529c11e74

Reply:
Fixed in bfe74d7